Orange/widgets/regression/owunivariateregression.py

```diff
@@ -17,13 +19,19 @@
 
 
 class OWUnivariateRegression(OWBaseLearner):
-    name = "Univariate Regression"
+    name        = "Univariate Polynomial Regression"
     description = "Univariate regression with polynomial expansion."
-    icon = "icons/UnivariateRegression.svg"
+    icon        = "icons/UnivariateRegression.svg"
 
-    LEARNER = LinearRegressionLearner
+    inputs = [("Learner", Learner, "set_learner"),
+              ("Preprocessor", Preprocess, "set_preprocessor"),
+              ("Learner", Learner, "set_learner")]
+
+    outputs = [("Learner", Learner),
+               ("Predictor", LinearModel),
+               ("Coefficients", Table)]
```

Comment:
`OWBaseLearner` already defines Learner and Predictor outputs. See its metaclass.